docs/docs/operator/runtime-threat-detection.md

```diff
@@ -0,0 +1,82 @@
+# Runtime Threat Detection & Response
+The runtime threat detection & response feature of the Kubescape operator enables users to have visibility into the runtime environment and respond to threats in real-time.
+Using eBPF probes toghether with Kubernetes context, the node agent effectively monitors the runtime environment for any suspicious activity.
+
+## How it works
+The runtime threat detection & response feature is divided into two main components: <br>
```

Comment:
This could be problematic for the lay-reader, because the two main components described are about threat detection and nothing about response. 
There needs to be some sort of forward looking or qualiying statement around the response piece as well as some clarification that threat detction has two main components.